haystack/components/evaluators/context_relevance.py

```diff
@@ -124,15 +111,15 @@ def __init__(
             Whether to raise an exception if the API call fails.
 
         """
+
         self.instructions = (
-            "Your task is to judge how relevant the provided context is for answering a question. "
-            "First, please extract statements from the provided context. "
-            "Second, calculate a relevance score for each statement in the context. "
-            "The score is 1 if the statement is relevant to answer the question or 0 if it is not relevant. "
-            "Each statement should be scored individually."
+            "Please extract only sentences from the provided context which are absolutely relevant and "
+            "required to answer the following question. If no relevant sentences are found, or if you "
+            "believe the question cannot be answered from the given context, return the phrase "
+            '"Insufficient Information"'
```

Comment:
Isn't the LLM instructed to return JSON strings? We can just ask it to return an empty list instead of this string.

Reply:
This was the requirements of the issue. I don't have a strong opinion on this, I'm OK with either empty list or this string. @mrm1001 your call.

Reply:
Please go with an empty list.

Reply:
We will need to run tests on a dataset to see if this matters. I tested the ragas prompt which uses this expression.

Reply:
I've just updated the PR to use an empty list. @mrm1001 from your thumbs up I gather you agree with Madeesh, right? 

What you mean by `We will need to run tests on a dataset to see if this matters.` ?

Reply:
I just meant that I ran the ragas prompt on the ARAGOG dataset, and they use a default string instead of empty list. I haven't tested it with a default output of empty list. I don't see how this would negatively impact the results, but we should still need to check on a dataset after this PR gets merged.